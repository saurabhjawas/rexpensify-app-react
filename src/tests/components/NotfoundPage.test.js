import React from 'react'
import { shallow } from 'enzyme'

import NotfoundPage from '../../components/NotfoundPage'

test('should generate a not found page component', () => {
  const wrapper = shallow(<NotfoundPage />)
  expect(wrapper).toMatchSnapshot()
})