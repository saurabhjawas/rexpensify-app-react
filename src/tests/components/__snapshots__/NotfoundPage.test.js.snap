// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should generate a not found page component 1`] = `
<div>
  404! - 
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
